src/cdk-experimental/menu/menu-bar.spec.ts

```diff
@@ -67,6 +70,77 @@ describe('MenuBar', () => {
       expect(spy).toHaveBeenCalledWith(menuItems[0]);
     });
   });
+
+  describe('with submenu', () => {
+    let fixture: ComponentFixture<MenuBarWithMenus>;
+
+    let menus: CdkMenu[];
+    let triggers: CdkMenuItemTrigger[];
+
+    /** opens all menus and asserts they are visible */
+    const openAllMenus = () => {
+      triggers[0].toggle();
+      detectChanges();
+
+      expect(menus.length).toEqual(1);
```

Comment:
I don't know how I feel about this assertion. Usually, I prefer to only include assertions in a test that are specific to what's being tested in that particular test. However, it's not like this is cluttering up the test code (since it's in another function), so I'll leave it up to you.